integr8ly/manifests
# INTLY-3286: Bump integreatly and solution explorer operator versions

## What
Adds CSV for integreatly operator 1.9.5:
- [x] Add integreatly operator 1.9.5 CSV
- [x] Bump integreatly operator version in package file.
- [x] Bump integreatly operator version in Makefile

Adds CSV for solution explorer operator 0.0.31:
- [x] Add solution explorer operator 0.0.31 CSV
- [x] Bump solution explorer operator version in package file.
- [x] Bump solution explorer operator version in Makefile

## Verification
1. Push the manifest to an app registry
2. Create an operator source which has this csv to a cluster.
3. Ensure that the Integreatly and tutorial web app operator can be installed from the Operator Hub

## Related PR
https://github.com/integr8ly/integreatly-operator/pull/125
